lib/order.rb

```diff
@@ -0,0 +1,43 @@
+class Order
+
+def initialize
+  @list_of_dishes = {"pizza"=>10, "pasta"=>8, "garlic bread"=>5}
+  @order = {}
+end
+
+def display_menu
+  p @list_of_dishes
+end
+
+def add_to_order(item)
+  item.each do |key, value|
+  @order[key] = value
+  end
+end
+
+def display_order
+  p @order
+  p @order.length
+end
```

Comment:
Really nit-picky but, your code is good and I don't have much to say!! I believe it is best practice to have defined indentation